pkgs/top-level/all-packages.nix

```diff
@@ -62,17 +62,17 @@ in
 
   inherit (lib) lowPrio hiPrio appendToName makeOverridable;
 
-  # Applying this to an attribute set will cause nix-env to look
-  # inside the set for derivations.
-  recurseIntoAttrs = attrs: attrs // { recurseForDerivations = true; };
+  # Make various Nix tools consider the contents of the resulting
+  # attribute set when looking for what to build, find, etc.
+  inherit (lib) recurseIntoAttrs;
 
   # This is intended to be the reverse of recurseIntoAttrs, as it is
```

Comment:
Shouldn't this comment be moved to `lib/attrsets.nix` (and adjusted) as well?

Reply:
It was already there, but I'll move this to the manual instead.

Reply:
I haven't touched the `pkgs.dontRecurseIntoAttrs` doc because it adds meaning that isn't in `lib`.